databroker/src/open_telemetry.rs

```diff
@@ -0,0 +1,25 @@
+#[cfg(feature="otel")]
+use {
+ opentelemetry::{KeyValue, runtime},
+ opentelemetry::sdk::{Resource, trace},
+ opentelemetry::trace::TraceError,
+ opentelemetry_otlp::WithExportConfig,
+};
+
+#[cfg(feature="otel")]
+pub fn init_trace() -> Result<trace::Tracer, TraceError> {
+    opentelemetry_otlp::new_pipeline()
+        .tracing()
+        .with_exporter(
+            opentelemetry_otlp::new_exporter()
+                .tonic()
+                .with_endpoint("http://localhost:4317"),
```

Comment:
Is 4317 a standard port for this type of functionality?

Reply:
Yes, Its a standard port for Jaeger backend. As Jaeger backend can also run on other device, I have made the endpoint configurable through env variable. 

This latest change will be part of my new signed PR (As this current PR is not signed, I was requested by Lukas to create a new signed PR)